src/index.html

```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="/layout_moyo-header/src/images/logo.png"
+        alt="logo"
+        class="logo-images"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="is-active nav-link">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              data-qa="hover"
+              class="nav-link"
+            >Laptops &amp; Computers</a>
```

Comment:
```suggestion
            >
              Laptops &amp; Computers
            </a>
```

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/147234742/e0906950-a906-4560-96d4-316ffb3eb1ae)
I already told you that it stops working as soon as I do that